docs/content/development/select-query.md

```diff
@@ -25,6 +26,7 @@ There are several main parts to a select query:
 |queryType|This String should always be "select"; this is the first thing Druid looks at to figure out how to interpret the query|yes|
 |dataSource|A String or Object defining the data source to query, very similar to a table in a relational database. See [DataSource](../querying/datasource.html) for more information.|yes|
 |intervals|A JSON Object representing ISO-8601 Intervals. This defines the time ranges to run the query over.|yes|
+|descending|Whether to make descending ordered result. Default is `false`(ascending).|no|
```

Comment:
Also, while paging the client increments the offset by 1 and queries again for next set of results, 
is it true for desending queries also or the client needs to decrement the offset by 1 ?  
This needs to be made clear in the docs.